### PR TITLE
README for API folder, validate API example

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
- Adds README to API folder to let people know it's now auto-generated
- Adds validate API example to overview section of docs

### Motivation
API page relaunch 🥳

### CODE REVIEW

File: content/en/api/v2/_index.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ The Datadog API uses resource-oriented URLs, uses status codes to indicate the s
 
 All requests to Datadog's API must be authenticated. Requests that write data require _reporting access_ and require an `API key`. Requests that read data require _full access_ and also require an `application key`.
 
+```bash
+curl "https://api.datadoghq.com/api/v1/validate" \
```
Comment (zbayoff) [inline content/en/api/v2/_index.md:21]: should be v2 here right?
```suggestion
curl "https://api.datadoghq.com/api/v2/validate" \
```

File: content/en/api/_index.md [hunk 2]
```diff
@@ -13,11 +13,16 @@ The Datadog API uses resource-oriented URLs, uses status codes to indicate the s
...
 
 All requests to Datadog's API must be authenticated. Requests that write data require _reporting access_ and require an `API key`. Requests that read data require _full access_ and also require an `application key`.
 
+```bash
+curl "https://api.datadoghq.com/api/v1/validate" \
+    -H "DD-API-KEY: <DATADOG_API_KEY>" \
+    -H "DD-APPLICATION-KEY: <DATADOG_APPLICATION_KEY>"
+```
+
 **Note**: All Datadog API Clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or overide this value directly when creating your Client.
```
Comment (sarina-dd) [inline content/en/api/_index.md:26]:
```suggestion
**Note**: All Datadog API Clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or overide this value directly when creating your client.
```

File: content/en/api/v1/_index.md [hunk 3]
```diff
@@ -18,6 +18,12 @@ The Datadog API uses resource-oriented URLs, uses status codes to indicate the s
 
 All requests to Datadog's API must be authenticated. Requests that write data require _reporting access_ and require an `API key`. Requests that read data require _full access_ and also require an `application key`.
 
+```bash
+curl "https://api.datadoghq.com/api/v1/validate" \
+    -H "DD-API-KEY: <DATADOG_API_KEY>" \
+    -H "DD-APPLICATION-KEY: <DATADOG_APPLICATION_KEY>"
+```
+
 **Note**: All Datadog API Clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or overide this value directly when creating your Client.
```
Comment (sarina-dd) [inline content/en/api/v1/_index.md:27]:
```suggestion
**Note**: All Datadog API clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or override this value directly when creating your client.
```

File: content/en/api/_index.md [hunk 4]
```diff
@@ -13,12 +13,17 @@ The Datadog API uses resource-oriented URLs, uses status codes to indicate the s
...
 All requests to Datadog's API must be authenticated. Requests that write data require _reporting access_ and require an `API key`. Requests that read data require _full access_ and also require an `application key`.
 
-**Note**: All Datadog API Clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or overide this value directly when creating your Client.
+```bash
+curl "https://api.datadoghq.com/api/v1/validate" \
+    -H "DD-API-KEY: <DATADOG_API_KEY>" \
+    -H "DD-APPLICATION-KEY: <DATADOG_APPLICATION_KEY>"
+```
+
+**Note**: All Datadog API clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or override this value directly when creating your client.
```
Comment (cswatt) [inline content/en/api/_index.md:26]:
```suggestion
**Note**: All Datadog API clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or override this value directly when creating your client.
```

File: content/en/api/v1/_index.md [hunk 5]
```diff
@@ -18,7 +18,13 @@ The Datadog API uses resource-oriented URLs, uses status codes to indicate the s
...
 All requests to Datadog's API must be authenticated. Requests that write data require _reporting access_ and require an `API key`. Requests that read data require _full access_ and also require an `application key`.
 
-**Note**: All Datadog API Clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or overide this value directly when creating your Client.
+```bash
+curl "https://api.datadoghq.com/api/v1/validate" \
+    -H "DD-API-KEY: <DATADOG_API_KEY>" \
+    -H "DD-APPLICATION-KEY: <DATADOG_APPLICATION_KEY>"
+```
+
+**Note**: All Datadog API clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, Set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or override this value directly when creating your client.
```
Comment (cswatt) [inline content/en/api/v1/_index.md:27]:
```suggestion
**Note**: All Datadog API clients are configured by default to consume Datadog US site APIs. If you are on the Datadog EU site, set the environment variable `DATADOG_HOST` to `https://api.datadoghq.eu` or override this value directly when creating your client.
```

**Pull request merged**